dashboard/views.py

```diff
@@ -38,10 +38,8 @@
 # string for resource type
 RESOURCE_TYPE_STRING = "resource_type"
 RESOURCE_VALUES = settings.RESOURCE_VALUES
-INVERSE = {}
-for key, value in RESOURCE_VALUES.items():
-    for resource_type in value['types']:
-        INVERSE[resource_type] = key
+RESOURCE_VALUES_MAP = settings.RESOURCE_VALUES_MAP
+RESOURCE_ACCESS_CONFIG = settings.RESOURCE_ACCESS_CONFIG
 
 # Is courses_enabled api enabled/disabled?
 COURSES_ENABLED = settings.COURSES_ENABLED
```

Comment:
If we have this, do we need the import on line 24?

Reply:
I'll put them all in the import